mantid-developer-linux.yml

```diff
@@ -0,0 +1,68 @@
+name: mantid
+
+channels:
+  - conda-forge
+  - DLR-SC # Required for OpenCascade
+
+dependencies:
+  - python=3.8.*
```

Comment:
Can we alphabetize within sections and split sections with a blank line? Maybe just simple sections like the common packages together then underneath os-specific sections. I'm not sure we need to distinguish between Python/C++ packaging as sections...

Reply:
Done :+1: